package.cordovabuild.json

```diff
@@ -109,7 +109,6 @@
     "angular": "1.6.7",
     "angular-animate": "1.6.7",
     "angular-local-storage": "^0.7.1",
-    "angular-nvd3": "^1.0.7",
```

Comment:
The speed graph should be fairly easy to add because we made `LineChart` in such a way that it can be easily re-used.
If we want, we could display plenty of other interesting things there too.